packages/manager/modules/manager-pci-common/src/api/data/regions.ts

```diff
@@ -21,3 +25,27 @@ export const getProjectRegions = async (
   });
   return data;
 };
+
+export const getS3StorageRegions = async (
+  regions: TRegion[],
+): Promise<TRegion[]> =>
+  regions.filter(({ services }) =>
+    services.find(({ name }) => S3_REGION_CAPACITY.includes(name)),
```

Comment:
To respect SRP and inversion of dependencies principles, I'll suggest to limit API functions to retrieve only DTO and let hooks that contains business logic to handle computations.
By doing this, we make code reusable and not dependent from the UI